packages/web/index.js

```diff
@@ -82,6 +81,13 @@ module.exports = (neutrino, opts = {}) => {
     );
   }
 
+  if ('manifest' in options) {
+    throw new ConfigurationError(
```

Comment:
I don't suppose you could add a test for this? Similar to:
https://github.com/neutrinojs/neutrino/blob/21225bfae3d83ca6e687858f246d8a4f883399d9/packages/web/test/web_test.js#L257-L311

Reply:
@edmorley added test!